lib/expandable.dart

```diff
@@ -163,6 +174,20 @@ class _ExpandablePanelState extends State<ExpandablePanel> {
 
   @override
   Widget build(BuildContext context) {
+    CrossAxisAlignment headerAlignment;
```

Comment:
instead of a temporary variable, make a function that returns a `CrossAxisAlignment` given a `ExpandablePanelHeaderAlignment`. It would reduce code size since you can use a return statements in the `switch()`.